### PR TITLE
feat(lights): improved exterior lights

### PR DESCRIPTION
**Pull request by Snapmatics**
## Summary of Changes

The following PR changes the overall exterior lighting of the aircraft.

## Exterior Lights Improvements
- [x] Take Off
- [x] Landing
- [x] Taxi
- [x] Runway
- [x] Strobe
- [x] Wing
- [x] Logo

## Take Off Lights
- Increased Range
- Increased Brightness
- Fixed Volumetricness
- Fixed (Inner/Outer/Position/Rotation)
> No Cockpit/Cabin Bleeding

Microsoft Flight Simulator 1_26_2021 4_32_39 AM

## Landing Lights
- Decreased Range
- Increased Brightness
- Fixed Volumetricness
- Fixed (Inner/Outer/Position/Rotation)
> No Cockpit/Cabin Bleeding

Microsoft Flight Simulator 1_26_2021 4_32_56 AM

## Take Off Lights + Landing Lights
Microsoft Flight Simulator 1_26_2021 4_37_10 AM

## Taxi Lights
- Decreased Range
- Increased Brightness
- Fixed Volumetricness
- Fixed (Inner/Outer/Position/Rotation)
> No Cockpit/Cabin Bleeding

Microsoft Flight Simulator 1_26_2021 4_33_42 AM

## Runway Lights
- Decreased Range
- Increased Brightness
- Fixed Volumetricness
- Fixed (Inner/Outer/Position/Rotation)
> No Cockpit/Cabin Bleeding

Microsoft Flight Simulator 1_26_2021 4_35_38 AM

## Taxi Lights + Runway Lights
Microsoft Flight Simulator 1_26_2021 4_37_57 AM

## Strobe Lights
- Increased Brightness (+40%)
- Fixed Volumetricness
> No Cockpit/Cabin Bleeding

Strobe

## Logo Lights
- Increased Brightness
- Fixed Rotation
> No Cockpit/Cabin Bleeding

image

## Wing Lights
- Increased Brightness
- Fixed Position
- Fixed Rotation
- Fixed Volumetricness
> No Cockpit/Cabin Bleeding

image

## Approaching Runway (Landing Video)
- Increased Range
- Brighter Runway

ezgif-1-35c28d590027
 

## References
[A320neo Exterior Lights](https://www.youtube.com/watch?v=apiD-YMJnoY)
Reference Screenshot_247

## Additional Context

Discord Username: `Snapmatics#0001`

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/systems.cfg [hunk 1]
```diff
@@ -45,6 +45,15 @@ lightdef.38 = Type:12#Index:1#LocalPosition:38.5,-1.825,5.980#LocalRotation:104,
 lightdef.39 = Type:12#Index:2#LocalPosition:38.5,1.760,5.980#LocalRotation:104,0,90#EffectFile:LIGHT_A32NX_ScreenOrange#Node:#PotentiometerIndex:11
 lightdef.40 = Type:13#Index:2#LocalPosition:35.0,0.960,9.000#LocalRotation:110,90,0#EffectFile:LIGHT_A32NX_CockpitSpotNarrow#PotentiometerIndex:97
 lightdef.41=  Type:13#Index:2#LocalPosition:34.488,-1.160,9.040#LocalRotation:110,90,0#EffectFile:LIGHT_A32NX_CockpitSpotNarrow#PotentiometerIndex:96
+//VOLUMETRIC
```
Comment (Benjozork) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/systems.cfg:48]: Comments are not valid in these files.

**Review by Benjozork — APPROVED** [review]
LGTM

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/systems.cfg (hunk 1 above)
Comment (Snapmatics) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/systems.cfg:48]: Thanks for letting me know, fixing.

File: A32NX/effects/LIGHT_A32NX_LeftRunway(V).fx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+[LightAttributes.0]
+Type=spot
+Size=0.0
+Range=80
+Intensity=100
+Softness=0.0
+SpotInner=0
+SpotOuter=35
+Volumetric=1
+ScatDir=0.0
```
Comment (pepperoni505) [inline A32NX/effects/LIGHT_A32NX_LeftRunway(V).fx:76]:
```suggestion
ScatDir=0.0

```

**Comment by marcman86** [comment]
There are two Artifacts. Which one to test? Can you clean this up?

**Comment by pepperoni505** [comment]
@marcman86 That should be fixed - let me know if it isn't

**Comment by marcman86** [comment]
Name: marcman86#4907
Date of testing: 20.2.2021
Version of the sim: 1.13.16.0
PR Tested: #3194
Tier of testing: 2
Changes to observe: Exterior lights
Testing technique: Tryout

Steps made to test the Pull Request:

1.) Spawned at night C&D EDDP
2.) Get bus to start
3.) Tested the lights (Image1+2)
4.) Took off (Image3)

Media:


Image1


Image2


Image3

Tested: 
- No bleeding in cockpit
- Logo lights go off with flaps 0
- Tested volumetricness


Issues: 
- Wing ice light turnes not on
- Taxi TO light is brighter than landing light, is this correct? (Image1)

Overall Rating: -

Conclusions of the testing: Looks good. Needs small fixes.

Hope this helps :-)

**Comment by Snapmatics** [comment]
> 
> 
> Name: marcman86#4907
> Date of testing: 20.2.2021
> Version of the sim: 1.13.16.0
> PR Tested: #3194
> Tier of testing: 2
> Changes to observe: Exterior lights
> Testing technique: Tryout
> 
> Steps made to test the Pull Request:
> 
> 1.) Spawned at night C&D EDDP
> 2.) Get bus to start
> 3.) Tested the lights (Image1+2)
> 4.) Took off (Image3)
> 
> Media:
> 
> 
> Image1
> 
> 
> Image2
> 
> 
> Image3
> 
> Tested:
> 
>     * No bleeding in cockpit
> 
>     * Logo lights go off with flaps 0
> 
>     * Tested volumetricness
> 
> 
> Issues:
> 
>     * Wing ice light turnes not on
> 
>     * Taxi TO light is brighter than landing light, is this correct? (Image1)
> 
> 
> Overall Rating: -
> 
> Conclusions of the testing: Looks good. Needs small fixes.
> 
> Hope this helps :-)

Correct, TO is brighter.

**Comment by FlightSimmer7700** [comment]
Category: QA Tester
Name: FlightSimmer7700#3468
Date of testing: 21/2/21
Version of the sim: 1.13.16.0
PR Tested: #3194 
Tier of testing: 2
Changes to observe: Exterior lights

Testing technique: Did a full flight and looked at the lights
Steps you've made to test the Pull Request:
1. Spawned in 
2. Looked at all lights and checked they work as expected
3. did a full flight

Media: Insert screenshots, recordings, or anything relevant to the review here.
image


Issues (If Any): Strobe lights on both sides are "double" meaning both the spare and normal strobes are on. As far as I know only one of those should be on so it would be better if one was removed on each side

Overall Rating: To be improved
Conclusions of the testing: Overall the lights are really nice with only the strobes to be fixed

**Comment by FlightSimmer7700** [comment]
Sorry I forgot to mention the wing light doesn't work as mentioned above by @marcman86

**Comment by Snapmatics** [comment]
> 
> 
> Sorry I forgot to mention the wing light doesn't work as mentioned above by @marcman86

Will fix that.

**Comment by beheh** [comment]
Removing Testing Started for now until the mentioned fix above is in.

**Comment by FlightSimmer7700** [comment]
Category: QA Tester
Name: FlightSimmer7700#3468
Date of testing: 26/2/21
Version of the sim: 1.13.16.0
PR Tested: #3194 
Tier of testing: 1
Changes to observe:
Fixed strobes and wing lights

Testing technique: Loaded in and looked
Steps you've made to test the Pull Request:
1. Loaded in
2. Looked at strobes
3. Looked at wing lights

Issues (If Any): None

Overall Rating: Good
Conclusions of the testing: LGTM

**Comment by ExampleWasTaken** [comment]
Category: QA Tester
Name: ExampleWasTaken#0886
Date of testing: 26/02/2021
Version of the sim: 1.13.16.0
PR Tested: #3194
Tier of testing: 2
Changes to observe: Improved exterior lights

Testing technique:
1. Loaded in C&D at NZAA
2. Powered up
3. Taxiied to RWY
4. Checked all modified lights

Flight notes (When done a full flight): N/A

Media: N/A

Issues (If Any): N/A

Overall Rating: Good
Conclusions of the testing: LGTM, much better than before!

**Comment by marcman86** [comment]
Category: QA Tester
Name: marcman86#4907
Date of testing: 26.2.2021
Version of the sim: 1.13.16.0
PR Tested: #3194
Tier of testing: 2
Changes to observe: Ext. lights
Testing technique: Tryout

Steps made to test the Pull Request:

1.) Spawned runway at EDDP
2.) Looked for ext. lights
3.) Took off to test logo light and volumetricness

- Strobe light cone fixed
- Wing light fixed

Media:


Image1


Image2


Image3

Issues: Beacon lights are only lights without volumetricness

Overall Rating: Good

Conclusions of the testing: Looks awesome. Maybe add two cones for the beacon lights?

Hope thi

**Comment by Snapmatics** [comment]
> 
> 
> Category: QA Tester
> Name: marcman86#4907
> Date of testing: 26.2.2021
> Version of the sim: 1.13.16.0
> PR Tested: #3194
> Tier of testing: 2
> Changes to observe: Ext. lights
> Testing technique: Tryout
> 
> Steps made to test the Pull Request:
> 
> 1.) Spawned runway at EDDP
> 2.) Looked for ext. lights
> 3.) Took off to test logo light and volumetricness
> 
>     * Strobe light cone fixed
> 
>     * Wing light fixed
> 
> 
> Media:
> 
> 
> Image1
> 
> 
> Image2
> 
> 
> Image3
> 
> Issues: Beacon lights are only lights without volumetricness
> 
> Overall Rating: Good
> 
> Conclusions of the testing: Looks awesome. Maybe add two cones for the beacon lights?
> 
> Hope this helps :-)

That will happen in the future, bleeding will occur if I do that.

**Pull request merged**